### DISCUSSION TITLE
Tree shaking with ES imports doesn't seem to work in v2.27.0

### DISCUSSION THREAD

**Discussion by ghost** [js-fns/date-fns#2827] [Q&A] [answered]
Or I am missing something.

This is when using
```ts
import format  from 'date-fns/format';
```

![image](https://user-images.githubusercontent.com/82456785/145034192-928cdd7d-8d7b-4ec0-b73b-1a31df9b523c.png)

And this is **the same** when using
```ts
import { format }  from 'date-fns';
```

![image](https://user-images.githubusercontent.com/82456785/145034336-8e35611e-b725-46b3-a9a3-65d67c352483.png)

**Comment by fturmel** [comment, marked as answer]
Seems fine. Have a look at this previous discussion, I think you might just be misunderstanding the webpack bundle analyzer values. https://github.com/date-fns/date-fns/discussions/2521#discussioncomment-891247
